spec/javascripts/models/beta/story_spec.js

```diff
@@ -686,4 +687,30 @@ describe('Story model', function () {
       expect(Story.releaseIsLate(story)).toBe(false);
     });
   });
+
+  describe('cloneStory', () => {
+    it('retuns a new story with null id', () => {
```

Comment:
Can we add a test that checks that everything else is equal to the original story?

Also, we should add tests to show that notes and tasks are not cloned as well, at least to have it documented.